docs/api-reference/gltf/gltf-loader.md

```diff
@@ -0,0 +1,51 @@
+# GLTFLoader
+
+Parses a glTF file into a hierarchical scenegraph description that can be used to instantiate an actual Scenegraph in most WebGL libraries. Can load both binary `.glb` files and JSON `.gltf` files.
+
+Also, certain glTF extensions can be fully or partially processed during loading. See [glTF Extensions](docs/api-reference/gltf-loaders/gltf-extensions.md).
+
+| Loader                | Characteristic  |
+| --------------------- | --------------- |
+| File Extensions       | `.glb`, `.gltf` |
+| File Types            | Binary, JSON, Linked Assets |
+| Input Format          | [glTF](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0) |
```

Comment:
I think the use of `input format` and `output format` for both loaders and writers is confusing , especally as they change depending on which one is used.

`File Format` and `Data Format` or `In-Memory Format` would be better.